mpy-cross/mpconfigport.h

```diff
@@ -68,16 +68,6 @@
 
 #define MICROPY_PY_BUILTINS_STR_UNICODE (1)
 
-// Define to 1 to use undertested inefficient GC helper implementation
-// (if more efficient arch-specific one is not available).
-#ifndef MICROPY_GCREGS_SETJMP
-    #ifdef __mips__
-        #define MICROPY_GCREGS_SETJMP (1)
-    #else
-        #define MICROPY_GCREGS_SETJMP (0)
-    #endif
-#endif
```

Comment:
Is there a reason to remove this?  I guess it matters for building mpy-cross on mips...

Reply:
I guess there's no reason to remove it. Added back.

Reply:
It's probably a cut-and-paste from the unix port that's not needed, but in the interest of making a minimal diff I think it should stay.